### PR TITLE
feat(iac): iac improvement

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
IaC improvement

## Resolution
* Environmental variables declaration 
* IaC improvement
* GHA CICD pipeline improvement

## Miscellaneous
* Removed `api-test` due to failing tests, these tests have been rectified in another PR.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
+# SWAGGER
 SWAGGER_USER=
 SWAGGER_PASSWORD=
 
 # API
 API_KEY=
 
-# Microsoft Graph
-GRAPH_BASE_URL=
+# APIM MDM
```
Comment (IgnatG) [inline .env.sample:16]: We shouldn't need MDM here

File: .github/workflows/README.md [hunk 2]
```diff
@@ -2,52 +2,123 @@
...
-* API tests
-* End to end (E2E) tests.
+##### 2. MDM 📦️
 
-The workflow consists of four jobs:
+- This job represents the deployment of the MDM (Master Data Management) micro-service.
+- Depends on the successful completion of the **Setup** job.
+- Environment: Uses the `environment` output from the **Setup** job.
+- Environment Variables:
+  - `NAME` 📁: Represents the name of the micro-service ("mdm").
```
Comment (IgnatG) [inline .github/workflows/README.md:80]: Should this be eStore?

File: .github/workflows/README.md [hunk 3]
```diff
@@ -2,52 +2,123 @@
...
+  - `environment`: Contains the value of the `ENVIRONMENT` environment variable.
+  - `timezone`: Contains the value of the `TIMEZONE` environment variable.
+- Steps:
+  - Environment 🧪: Displays the environment set to the `ENVIRONMENT` value.
+  - Timezone 🌐: Displays the timezone set to the `TIMEZONE` value.
 
-* Unit tests
-* API tests
-* End to end (E2E) tests.
+##### 2. MDM 📦️
```
Comment (IgnatG) [inline .github/workflows/README.md]: eStore?

File: .github/workflows/README.md [hunk 4]
```diff
@@ -2,52 +2,123 @@
...
 
-The workflow consists of four jobs:
+- This job represents the deployment of the MDM (Master Data Management) micro-service.
```
Comment (IgnatG) [inline .github/workflows/README.md]: eStore?

Comment (abhi-markan) [reply .github/workflows/README.md]: Have replaced all MDM instances to ESTORE.

File: .env.sample (hunk 1 above)
Comment (abhi-markan) [reply .env.sample:16]: I have been informed number generator call is required for eStore site creation.

Comment (IgnatG) [reply .env.sample:16]: Oh! Makes sense then.

**Review by IgnatG — DISMISSED** [review]
LGTM

**Review by lmarrai-sw — DISMISSED** [review]

**Review by lmarrai-sw — DISMISSED** [review]

**Review by IgnatG — APPROVED** [review]

**Pull request merged**